app/lib/fsHelper.js

```diff
@@ -18,7 +28,7 @@ function saveJsonSync(obj, filename) {
 
 function loadJsonSync(filename) {
   const path = `${OUTPUT_DIR}/${filename}.json`;
-  const jsonString = fs.existsSync(path) ? fs.readFileSync(path, 'utf8') : undefined;
+  const jsonString = fileExists(path) ? fs.readFileSync(path, 'utf8') : undefined;
```

Comment:
It was pointed out in the PR for the POMI-DATA-ETL that fs.existsSync has  been deprecated, and should be changed.

The methods that replace `exists`, namely `stat` and `access`, both throw an exception, with the final recommendation that if you are going to use the file, just try and open it and catch the exception if it fails. Why you would care about the existence of a file if you weren't going to do something with it, I don't know.

`exists` has been replaced with something that adds much more complexity, and if we want to move on from using `exists` we need to replace it across all projects in a consistent manner, with the official line from Joyent being 'just try to use a file/folder and catch the error if there is one'. 

i.e. this will be a structural change, as there is no equivalent non-deprecated method that Joyent recommend you use to check if a file exists before using it.